index.html

```diff
@@ -11,6 +11,18 @@
 </head>
 
 <body>
+    <nav>
+        <ul>
+            <li><a href="#about">About</a></li>
+            <li><a href="#experience">Experience</a></li>
+            <li><a href="#skills">Skills</a></li> <!-- Skills section -->
+            <li><a href="#projects">Projects</a></li> <!--Projects section -->
+            <li><a href="#connect">Connect</a></li>
```

Comment:
In order to have a smooth scrolling when clicking on your nav links to their respective sections, you could add a smooth value to the scroll-behavior CSS property to your html selector in your css at the top:

html {
   scroll-behavior: smooth;
}